inventory/sample/group_vars/k8s_cluster/k8s-net-kube-ovn.yml

```diff
@@ -55,3 +55,9 @@ kube_ovn_enable_ssl: false
 
 ## dpdk
 kube_ovn_dpdk_enabled: false
+
+## enable interconnection to an existing IC database server.
+kube_ovn_ic_enable: false
+kube_ovn_ic_autoroute: true
+kube_ovn_ic_dbhost: "10.17.1.1"
+kube_ovn_ic_zone: "sample"
```

Comment:
Thanks @JochenFriedrich 
If the sample variables be same with the default vars, it maybe more clear :-)